docs/refmans/gui/viselements/generic/date_range.md_template

```diff
@@ -52,8 +52,8 @@ The date range selector looks like this:
 
 An `on_change` callback can be set to retrieve the user's selection:
 ```python
-def on_change(s: State, name: str, value: any):
-    if name == "dates":
+def on_change(s: State, var_name: str, var_value: any):
```

Comment:
Minor comment but might as well change `s` to `state` in this PR. I think this is the only occurrence as a callback function parameter that we abbreviate `state` as `s`. I prefer abbreviating it only in lambdas and in a context manager